src/Execution/ExecutionContextBuilder.php

```diff
@@ -1,100 +0,0 @@
-<?php
-
-namespace Digia\GraphQL\Execution;
-
-use Digia\GraphQL\Error\ExecutionException;
-use Digia\GraphQL\Language\Node\DocumentNode;
-use Digia\GraphQL\Language\Node\NodeKindEnum;
-use Digia\GraphQL\Language\Node\OperationDefinitionNode;
-use Digia\GraphQL\Schema\Schema;
-
-/**
- * Class ExecutionContextBuilder
- * @package Digia\GraphQL\Execution
- */
-class ExecutionContextBuilder
```

Comment:
@crisu83 I think to have this Builder is somehow easier to write unit test.

Reply:
There was no tests and removing the builder is consistent with rest of the project.

Reply:
I would still remove the builder because I also removed all the other context builders to reduce complexity.

Reply:
Also, I don't really see the need for unit testing the builder. We have quite a lot of "black-box" tests for this already.